cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/task/LoadMonitorTaskRunnerTest.java

```diff
@@ -57,11 +57,11 @@
  * The unit test for metric fetcher manager.
  */
 public class LoadMonitorTaskRunnerTest extends CCKafkaIntegrationTestHarness {
-  private static final long WINDOW_MS = 10000L;
+  private static final long WINDOW_MS = TimeUnit.SECONDS.toMillis(10);
   private static final int NUM_WINDOWS = 5;
   private static final int NUM_TOPICS = 100;
   private static final int NUM_PARTITIONS = 4;
-  private static final long SAMPLING_INTERVAL = 100000L;
+  private static final long SAMPLING_INTERVAL = TimeUnit.SECONDS.toMillis(100);
```

Comment:
(Not introduced in this patch): Can we rename this private variable to `SAMPLING_INTERVAL_MS`?